src/styles/settings.scss

```diff
@@ -111,4 +111,20 @@ $header-topbar-bg: #434343;
 $header-bg: #363636;
 
 //OTHER SETTINGS
-$base-padding: 24px;
\ No newline at end of file
+$base-padding: 24px;
+
+//Animations
+
+@mixin text-hover-animation {
+    transition-property: color;
+    transition-duration: 250ms;
+    transition-timing-function: linear;
+    transition-delay: 150ms;
+}
+
+@mixin bg-hover-animation {
+    transition-property: background-color;
+    transition-duration: 250ms;
+    transition-timing-function: linear;
+    transition-delay: 100ms;
+}
```

Comment:
Tutaj powtarza się wszystko oprócz transition-property wiec można by dodać dodatkowy mixin.

Reply:
To jest moim zdaniem ok ponieważ `Może potrzebować, żeby czas animacji koloru tła i czas animacji koloru tekstu/ikony były różne. `
Ale czasy przeniósłbym do zmiennych SCSS aby łatwo móc je zmieniać globalnie zgodnie z wymaganiem